sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/randomExpressions.scala

```diff
@@ -145,6 +156,11 @@ case class Randn(child: Expression) extends RDG with ExpressionWithRandomSeed {
   }
 
   override def freshCopy(): Randn = Randn(child)
+
+  override def flatArguments: Iterator[Any] = Iterator(child)
+  override def sql: String = {
+    s"randn(${if (useRandSeed) "" else child.sql})"
```

Comment:
The naming `useRandSeed` might be a little mismatched. This is used only here to hide random seed. Maybe, something like `hideSeed` is more direct?

Reply:
Currently, `Randn(child = expr, useRandSeed = true)` seems to be possible. It might look weird because it will not use rand seed.

Reply:
yea, the name looks reasonable.